zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, Set<String>> runtimeInfos;
```

Comment:
@cloverhearts  Your English is not bad 👍 

If i add `transient` modifier, when `Gson.toJson(Paragraph)` is done to send message  about note/paragraph, the `runtimeInfos` wont be serialized and will not be received by browser(`runtimeInfos` wont get persisted in `note.json`), 

Whereas if i add the modifier(current implementation), it will get persisted in JSON. But we dont want it, since it a run time information, and we might not want to persist in the `note.json`

Is there a way where i can serailize this info(for sending to client browser) as well as not persist in json?
PS: `runtimeInfos` object has the paragraph url information.